proto/umee/incentive/v1/query.proto

```diff
@@ -217,4 +231,29 @@ message QueryCurrentRatesResponse {
     (gogoproto.nullable)     = false,
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
   ];
+}
+
+// QueryLastRewardTime defines the request structure for the LastRewardTime gRPC service
+// handler.
+message QueryLastRewardTime {}
+
+// QueryLastRewardTimeResponse defines the response structure for the LastRewardTime gRPC
+// service handler.
+message QueryLastRewardTimeResponse {
+  int64 time = 1;
+}
+
+// QueryActualRates defines the request structure for the ActualRates gRPC service handler.
+message QueryActualRates {
+  // uToken is the uToken denomination whose current annual rate of rewards is being queried
+  string uToken = 1;
+}
+
+// QueryActualRatesResponse defines the response structure for the ActualRates gRPC service handler.
+message QueryActualRatesResponse {
+  // APY is the oracle price-adjusted APY of the bonded uToken.
+  string APY = 1 [
+    (gogoproto.nullable)   = false,
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"
+  ];
```

Comment:
```suggestion
    cosmos.base.v1beta1.DecCoin APY = 1 [(gogoproto.nullable)   = false;
```

Reply:
APY here is meant to be a percent (based on usd value), not token

Reply:
for more detail:

say bonding $100 of UMEE results in $10 of ATOM per year at current rates. The return for `ActualRates` is `0.1` (10%) whereas the return of `CurrentRates` specifies an UMEE reference amount and a resulting ATOM amount in tokens

Reply:
misread it, I thought you cast to DecCoin